globus_automate_client/action_client.py

```diff
@@ -176,15 +180,14 @@ def log(
         # *reverse_order* MUST BE None to prevent reversing the sort order.
         # Any other value, including False, will reverse the sort order.
```

Comment:
Is this comment still true? Was it ever?

Reply:
Yes, it is true. While developing #90 I found that `reverse_order` was extremely sensitive: `False` and `True` both resulted in a reversed sort order, perhaps because the values were interpreted as strings by the server. To work around this, the client was setting a default of `None` so that if the user didn't specify a reverse order, that default `None` would make its way in as the value that's passed here.

In #90 I adjusted the code so that it internally uses booleans for `reverse_order` everywhere and swaps out `False` for `None` at this point in the code. This comment is here to clarify why this value swap is happening.